README.md

```diff
@@ -1,24 +1,20 @@
 # MTProxy
 
-# Building
+## Building
+Install dependencies, you would need common set of tools for building from source, and source packages for `openssl` and `zlib`.
```

Comment:
Why "source packages" and not "development packages"? I think source packages stands for getting actual source from which the package was built from.

Reply:
@stek29 Debian site call them so: 
![image](https://user-images.githubusercontent.com/5345489/41118213-f29fbf08-6a97-11e8-9609-92367e2502a9.png)
I can rename it...

Reply:
openssl is the source package, not the libssl-dev, no?
and libssl-dev seems to be called "development files" too...

Reply:
@stek29 okay, I fixed it.